gapis/replay/builder/builder.go

```diff
@@ -603,20 +605,28 @@ func (b *Builder) Build(ctx context.Context) (protocol.Payload, ResponseDecoder,
 		log.I(ctx, "Resource count:         %d", len(payload.Resources))
 	}
 
-	// TODO: check that each Postback consumes its expected number of bytes.
-	responseDecoder := func(r io.Reader, err error) {
-		var d binary.Reader
-		if r != nil {
-			d = endian.Reader(r, byteOrder)
+	responseDecoder := func(pd *gapir.PostData) {
+		// TODO: should we skip it instead of return error?
+		ctx = log.Enter(ctx, "responseDecoder")
+		if pd == nil {
+			log.E(ctx, "Cannot handle nil PostData")
 		}
-		go func() {
-			for _, p := range b.decoders {
-				err = p(d, err)
-				if err != nil {
-					d = nil
+		crash.Go(func() {
```

Comment:
Not entirely sure about spawning a new go routine now. 
This function is called whenever a new post gRPC call is made, right? Assuming the rpc is non blocking, we can probably safely drop this.